### PR TITLE
About Page: remove submenu and add link to page in footer

### LINKED ISSUE #12315: Jetpack About Page: Remove sidebar link, Add footer link
Currently the "About Jetpack" link lives in the wp-admin sidebar.

Screen Shot 2019-05-08 at 5 04 39 PM

After chatting with Matt, he requested we move it into the footer of the Jetpack dashboard area, and out of the sidebar.

Currently:
Screen Shot 2019-05-08 at 5 03 15 PM

Proposed:
Screen Shot 2019-05-08 at 5 06 29 PM

1) Add "About" link to footer
2) Remove sidebar link to about page
3) Make "Automattic Airline" link in the footer also go to about page within wp-admin (currently takes the customer to automattic.com - I'd like to keep them in the admin

### PR DESCRIPTION
**Pull request by jeherve**
Fixes #12315

#### Changes proposed in this Pull Request:

You can now access the About Page from 4 different locations:

- The Automattic link at the bottom of the Jetpack dashboard pages.
- The About link at the bottom of the Jetpack dashboard pages.
- The Automattic logo on any Jetpack non-react admin pages (e.g. Settings > Sharing)
- The about link at the bottom of any Jetpack non-react admin pages (e.g. Settings > Sharing)

#### Testing instructions:

* Go to any of the pages I mentioned above?
* Do you see the links?
* Can you click on them, does the page open in the same tab?

**Note**: 
- If your site is connected to wpcom or in dev mode, the About link will lead to the in-plugin About page.
- If you are not connected yet, the About link will lead to Jetpack.com.

#### Proposed changelog entry for your changes:

* None

### CODE REVIEW

File: _inc/lib/admin-pages/class.jetpack-admin-page.php [hunk 1]
```diff
@@ -315,11 +315,14 @@ public static function wrap_ui( $callback, $args = array() ) {
...
 				<ul class="jp-footer__links">
 					<li class="jp-footer__link-item">
 						<a href="https://jetpack.com" target="_blank" rel="noopener noreferrer" class="jp-footer__link" title="<?php esc_html_e( 'Jetpack version', 'jetpack' ); ?>">Jetpack <?php echo JETPACK__VERSION; ?></a>
 					</li>
+					<li class="jp-footer__link-item">
+						<a href="<?php echo esc_url( admin_url() . 'admin.php?page=jetpack_about' ); ?>" title="<?php esc_attr__( 'About Jetpack', 'jetpack' ); ?>" class="jp-footer__link"><?php echo esc_html__( 'About', 'jetpack' ); ?></a>
```
Comment (eliorivero) [inline _inc/lib/admin-pages/class.jetpack-admin-page.php]:
```suggestion
						<a href="<?php echo esc_url( admin_url( 'admin.php?page=jetpack_about' ) ); ?>" title="<?php esc_attr__( 'About Jetpack', 'jetpack' ); ?>" class="jp-footer__link"><?php echo esc_html__( 'About', 'jetpack' ); ?></a>
```

The path should be inside `admin_url`. See https://codex.wordpress.org/Function_Reference/admin_url#Examples

**Review by eliorivero — CHANGES_REQUESTED** [review]
Minor change requested, added code suggestion

**Comment by eliorivero** [comment]
About page is no longer in the menu ✅ 

Captura de Pantalla 2019-05-09 a la(s) 15 42 48

It's in the footer and properly linked ✅ 

Captura de Pantalla 2019-05-09 a la(s) 15 43 05

Goes to blank page ❗️ 

Captura de Pantalla 2019-05-09 a la(s) 15 43 26

File: _inc/lib/admin-pages/class.jetpack-admin-page.php (hunk 1 above)
Comment (jeherve) [reply _inc/lib/admin-pages/class.jetpack-admin-page.php]: Good call. Done in 698087716a7481f72a21080e2f4e6cff4c3faba3

**Comment by jeherve** [comment]
Thanks for the feedback. This should be good for another review!

**Review by eliorivero — APPROVED** [review]

**Comment by eliorivero** [comment]
🐑

**Pull request merged**